controllers/user.js

```diff
@@ -4,7 +4,7 @@ import { User } from '../models';
 
 dotenv.config();
 const { JWT_SECRET } = process.env;
-const generateToken = id => jwt.sign({ id }, JWT_SECRET, { expiresIn: '24h' });
+const generateToken = (id, expiresIn = '24h') => jwt.sign({ id }, JWT_SECRET, { expiresIn });
```

Comment:
@chingsley what are you trying to achieve here?

Reply:
I implemented a 'rememberMe' functionality during sign in. If during sign in a user sets the 'rememberMe' property to true (i.e  "rememberMe": true ), the token generated will last for up to 240 hours before expiration, else, a default value of 24 hours is used so the token expires, by default,  after 24 hours'